### DISCUSSION TITLE
关于程序单例

### DISCUSSION THREAD

**Discussion by TogetherYear** [clash-verge-rev/clash-verge-rev#1668] [Q&A]
大佬好 我看代码中是用服务端口去判断有没有启动实例，是不是也是因为官方单例插件会导致tauri程序只能启动一个的原因。

**Comment by jerry-rs** [comment]
我感觉用服务端口去判断可能存在问题，理由如下：
1. 假设服务端口是被占用了，但是服务进程是僵死状态，那么就会存在问题
